crates/bevy_window/src/lib.rs

```diff
@@ -71,3 +73,6 @@ impl Plugin for WindowPlugin {
         }
     }
 }
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+pub struct ModifiesWindows;
```

Comment:
The system is called change_window, so maybe call this ChangesWindows

Reply:
My intent with that name was that it can be added to any system that modifies the `Windows` resource and not be specific to `change_window`. I didn't want to make it specific to `change_window` because I can't put it inside `bevy_winit`.

Reply:
Can systems have multiple labels, if not then that approach doesn’t really hold because a system could modify multiple things And the windows.